deps/wazuh_testing/wazuh_testing/tools/__init__.py

```diff
@@ -91,28 +91,34 @@
 
 
 def get_version():
+    try:
+        if platform.system() in ['Windows', 'win32']:
+            with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
+                version = f.read()
+                return version[:version.rfind('\n')]
 
-    if platform.system() in ['Windows', 'win32']:
-        with open(os.path.join(WAZUH_PATH, 'VERSION'), 'r') as f:
-            version = f.read()
-            return version[:version.rfind('\n')]
-
-    else:  # Linux, sunos5, darwin, aix...
-        return subprocess.check_output([
-          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"
-        ], stderr=subprocess.PIPE).decode('utf-8').rstrip()
+        else:  # Linux, sunos5, darwin, aix...
+            return subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-v"],
+                                           stderr=subprocess.PIPE).decode('utf-8').rstrip()
+    except Exception:
+        return 'N/A'
 
 
 def get_service():
-    if platform.system() in ['Windows', 'win32']:
-        return 'wazuh-agent'
-
-    else:  # Linux, sunos5, darwin, aix...
-        service = subprocess.check_output([
-          f"{WAZUH_PATH}/bin/wazuh-control", "info", "-t"
-        ], stderr=subprocess.PIPE).decode('utf-8').strip()
-
-    return 'wazuh-manager' if service == 'server' else 'wazuh-agent'
+    try:
+        if platform.system() in ['Windows', 'win32']:
+            if os.path.exists(WAZUH_PATH):
+                service = 'wazuh-agent'
+            else:
+                service = 'N/A'
+        else:  # Linux, sunos5, darwin, aix...
+            output = subprocess.check_output([f"{WAZUH_PATH}/bin/wazuh-control", "info", "-t"],
+                                             stderr=subprocess.PIPE).decode('utf-8').strip()
+            service = 'wazuh-manager' if output == 'server' else 'wazuh-agent'
+    except Exception:
+        service = 'N/A'
```

Comment:
Why are you returning 'N/A' and not False or something more descriptive? Could we choose a better way here?

Reply:
Refer to [this](https://github.com/wazuh/wazuh-qa/pull/3972#discussion_r1277367226) comment

Reply:
Ok